torch/optim/lr_scheduler.py

```diff
@@ -95,12 +97,12 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
-    def get_last_lr(self):
+    def get_last_lr(self) -> List[float]:
         """ Return last computed learning rate by current scheduler.
         """
         return self._last_lr
 
-    def get_lr(self):
+    def get_lr(self) -> List[float]:
```

Comment:
are all `get_lr` need to also have the return type specified?

Reply:
Yes, they should all have the same signature. I was going to leave that for a separate PR though. The workflow here is a first PR to move all existing annotations in stub files inline without changes (adding `ignore_errors=True` for the files in `mypy.ini` if needed), and then a second PR to fix errors and complete missing annotations.  Mixing fixes with moving code around makes PRs hard to review.

If something is failing in internal CI and adding a few more ` -> List[float]` annotations, then of course we can do that.